internal/services/pam/testdata/TestSetDefaultBrokerForUser/golden/set_default_broker_for_existing_user_with_no_broker/cache.db

```diff
@@ -0,0 +1,44 @@
+    "2222": '{"Name":"userwithinactivebroker","UID":2222,"GID":22222,"Gecos":"userwithinactivebroker","Dir":"/home/userwithinactivebroker","Shell":"/bin/dash","LastPwdChange":-1,"MaxPwdAge":-1,"PwdWarnPeriod":-1,"PwdInactivity":-1,"MinPwdAge":-1,"ExpirationDate":-1,"LastLogin":"ABCDETIME"}'
+    "3333": '{"Name":"userlocalbroker","UID":3333,"GID":33333,"Gecos":"userlocalbroker","Dir":"/home/userlocalbroker","Shell":"/bin/zsh","LastPwdChange":-1,"MaxPwdAge":-1,"PwdWarnPeriod":-1,"PwdInactivity":-1,"MinPwdAge":-1,"ExpirationDate":-1,"LastLogin":"ABCDETIME"}'
+    "4444": '{"Name":"usersetbroker","UID":4444,"GID":44444,"Gecos":"usersetbroker","Dir":"/home/usersetbroker","Shell":"/bin/dash","LastPwdChange":-1,"MaxPwdAge":-1,"PwdWarnPeriod":-1,"PwdInactivity":-1,"MinPwdAge":-1,"ExpirationDate":-1,"LastLogin":"ABCDETIME"}'
+    "5555": '{"Name":"userupdatebroker","UID":5555,"GID":55555,"Gecos":"userupdatebroker","Dir":"/home/userupdatebroker","Shell":"/bin/zsh","LastPwdChange":-1,"MaxPwdAge":-1,"PwdWarnPeriod":-1,"PwdInactivity":-1,"MinPwdAge":-1,"ExpirationDate":-1,"LastLogin":"ABCDETIME"}'
+UserByName:
+    userlocalbroker: '{"Name":"userlocalbroker","UID":3333,"GID":33333,"Gecos":"userlocalbroker","Dir":"/home/userlocalbroker","Shell":"/bin/zsh","LastPwdChange":-1,"MaxPwdAge":-1,"PwdWarnPeriod":-1,"PwdInactivity":-1,"MinPwdAge":-1,"ExpirationDate":-1,"LastLogin":"ABCDETIME"}'
+    usersetbroker: '{"Name":"usersetbroker","UID":4444,"GID":44444,"Gecos":"usersetbroker","Dir":"/home/usersetbroker","Shell":"/bin/zsh","LastPwdChange":-1,"MaxPwdAge":-1,"PwdWarnPeriod":-1,"PwdInactivity":-1,"MinPwdAge":-1,"ExpirationDate":-1,"LastLogin":"ABCDETIME"}'
+    userupdatebroker: '{"Name":"userupdatebroker","UID":5555,"GID":55555,"Gecos":"userupdatebroker","Dir":"/home/userupdatebroker","Shell":"/bin/zsh","LastPwdChange":-1,"MaxPwdAge":-1,"PwdWarnPeriod":-1,"PwdInactivity":-1,"MinPwdAge":-1,"ExpirationDate":-1,"LastLogin":"ABCDETIME"}'
+    userwithbroker: '{"Name":"userwithbroker","UID":1111,"GID":11111,"Gecos":"userwithbroker gecos\nOn multiple lines","Dir":"/home/userwithbroker","Shell":"/bin/bash","LastPwdChange":-1,"MaxPwdAge":-1,"PwdWarnPeriod":-1,"PwdInactivity":-1,"MinPwdAge":-1,"ExpirationDate":-1,"LastLogin":"ABCDETIME"}'
+    userwithinactivebroker: '{"Name":"userwithinactivebroker","UID":2222,"GID":22222,"Gecos":"userwithinactivebroker","Dir":"/home/userwithinactivebroker","Shell":"/bin/dash","LastPwdChange":-1,"MaxPwdAge":-1,"PwdWarnPeriod":-1,"PwdInactivity":-1,"MinPwdAge":-1,"ExpirationDate":-1,"LastLogin":"ABCDETIME"}'
+UserToBroker:
+    "1111": '"broker-id"'
+    "2222": '"inactive-broker-id"'
+    "4444": '"1902181170"'
+    "5555": '"tobereplaced-broker-id"'
```

Comment:
It was not updated?

Reply:
Can you re-read the name of this golden file?

Reply:
ooops, correct! Sorry